jolt-core/src/poly/pedersen.rs

```diff
@@ -55,12 +56,20 @@ impl<G: CurveGroup> PedersenGenerators<G> {
 }
 
 impl<G: CurveGroup> PedersenGenerators<G> where G::Affine: IntoSpartan {
-    pub fn into_spartan(&self) -> Vec<<G::Affine as IntoSpartan>::SpartanAffine> {
+    pub fn to_spartan(&self) -> Vec<<G::Affine as IntoSpartan>::SpartanAffine> {
```

Comment:
And we should only need one of `{HyraxCommitment, PedersenGenerators}::{to_spartan, to_spartan_bn_256}`